app.py

```diff
@@ -3,20 +3,24 @@
 from api.routes.api_routes import api
 from api.helpers.errors import APIError
 import traceback
-import pry
+import os
 
 app = Flask(__name__, template_folder = 'api/views')
 
-# app.config.from_envvar('ENV_FILE_LOCATION')
-# POTENTIAL FIRST SOLUTION
-import os
-app.config['MONGO_SETTINGS'] = {
-    'host': os.environ.get('MONGDB_URI')
-}
+"""For Production Only"""
+# # please comment out if not using
+
+# app.config['MONGODB_SETTINGS'] = {
```

Comment:
When we PR, we should be cautious to always ensure this is exposed and line 23 is commented out

Reply:
This is to prevent Heroku breaks